### ISSUE TITLE
How to get the validation_accuracy or validation_loss in the result of the iucnn_train_model (model="bnn-class") function?

### ISSUE THREAD

**Issue by Orange-chen-PEAR** [IUCNN/IUCNN#48]
Hi @ALL,

I had some trouble in bnn-class model. I randomly split the data set into a training set (70% of the entries) and a test set (30%). I found the validation_accuracy or validation_loss was NaN after the implement of the iucnn_train_model (model="bnn-class",dropout_rate=0.05) function.

I want to know the methods or references for determining the validation_accuracy or validation_loss in the iucnn_train_model (model="bnn-class") function.

Can you help me with this?
Thank you for your kind consideration of this request.

Jinnan
